src/swupd/manifest.go

```diff
@@ -39,9 +42,11 @@ type ManifestHeader struct {
 
 // Manifest represents a bundle or list of bundles (MoM)
 type Manifest struct {
-	Name   string
-	Header ManifestHeader
-	Files  []*File
+	Name         string
+	Header       ManifestHeader
+	Files        []*File
+	DeletedFiles []*File
+	Changed      bool
```

Comment:
What does Changed stands for?

Reply:
Looks like I don't use that like I thought I would. It was intended to be a boolean representing whether the manifest needed to be regenerated or not. I don't need that any more because I append changed files to the `newManifests` slice.